content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -0,0 +1,111 @@
+# Generate live (real-time) metrics and plots for experiments
+
+When you
+[submit a new experiment](/doc/studio/user-guide/projects-and-experiments/run-experiments)
+from Iterative Studio, the
+[CI/CD setup](/doc/use-cases/ci-cd-for-machine-learning) in your Git repository
+gets invoked. If this setup includes a model training process, it will be
+triggered.
+
+In this model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This will enable you to view all intermediate results in Studio
+while your experiment is still running.
+
+This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-the-updates)
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+TODO: Replace this with a video tutorial for live metrics and plots
+
+## Set up an access token
+
+Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
+to the metrics and plots.
+
+### Generate access token
+
+To set up the access token, open your user profile page. In the
+`Studio access token` section, click on `Generate new token`.
+
+### Regenerate access token
+
+You can regenerate the access token at any point. The old access token will no
+longer be authorized to send live metrics and plots to Iterative Studio.
+
+### Reset (delete) access token
+
+You can delete the access token when you no longer need it.
+
+The option to delete the access token is also available when you try to change
+your password. That is, you can reset all your access credentials (your password
+and the access token) at once. This is handy if you suspect that your account
+security may have been compromised.
+
+## Configure your model training CI job
+
+You should define the following environment variables in your CI job:
+
+- `STUDIO_REPO_URL`: URL for the Git repository connected to the project. E.g.,
+  `github:iterative/example-get-started`
+- `STUDIO_TOKEN`: Your access token. The access token must be present in any
+  request that sends data to the Iterative Studio ingestion endpoint. Requests
+  with missing or incorrect access tokens are rejected with an appropriate HTTP
+  error code and error message.
```

Comment:
We could even try to include the sentence about generating the token in here and completely avoid the previous H2 (**Set up an access token**).

Reply:
Didn't do this, coz I think the access token set up deserves explaining separately. But I've made that section (on access token setup) small, so it should be ok now.